### PR TITLE
{bio}[GCCcore/12.3.0] BioPerl v1.7.8

### PR DESCRIPTION
**Pull request by smoors**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by smoors** [comment]
@boegelbot: please test @ generoso

**Comment by smoors** [comment]
Test report by @smoors
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
node398.hydra.os - Linux Rocky Linux 8.8, x86_64, Intel(R) Xeon(R) Gold 6132 CPU @ 2.60GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/smoors/415ccc0855e6b99fcfbd84048a149033 for a full test report.

**Comment by smoors** [comment]
@boegelbot: please test @ generoso

**Comment by smoors** [comment]
@boegelbot please test @ jsc-zen2

File: easybuild/easyconfigs/x/XML-LibXML/XML-LibXML-2.0209-GCCcore-12.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    }),
+]
+
+modextrapaths = {
+    'PERL5LIB': 'lib/perl5/site_perl/%(perlver)s/',
+}
+
+sanity_check_paths = {
+    'files': [],
+    'dirs': ['lib/perl5/site_perl/%(perlver)s/x86_64-linux-thread-multi/XML/LibXML'],
```
Comment (boegel) [inline easybuild/easyconfigs/x/XML-LibXML/XML-LibXML-2.0209-GCCcore-12.3.0.eb:62]: @smoors Please use `%(arch)s` here (so it also works on Arm), and apply the same fix to the existing `XML-LibXML-2.0208-GCCcore-12.2.0.eb` where you probably inherited this from (in this PR is fine)

```suggestion
    'dirs': ['lib/perl5/site_perl/%(perlver)s/%(arch)s-linux-thread-multi/XML/LibXML'],
```

We already have this in all other `XML-LibXML` easyconfigs

**Review by boegel — CHANGES_REQUESTED** [review]

**Comment by smoors** [comment]
@boegelbot: please test @ generoso

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in total)
node3138.skitty.os - Linux RHEL 8.8, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/boegel/18504e4a50f7c582fb16387a70ae6f27 for a full test report.

**Review by boegel — APPROVED** [review]
lgtm

**Comment by boegel** [comment]
Going in, thanks @smoors!

**Pull request merged**